torch/_inductor/mkldnn_lowerings.py

```diff
@@ -588,6 +588,14 @@ def qlinear_unary(
                     torch.tensor(x_scale, dtype=torch.float32), name="x_scale"
                 )
             else:
+                x_scale_size = x_scale.get_size()
+                if len(x_scale_size) == 2:
+                    if x_scale_size[0] == 1 and x_scale_size[1] == 1:
```

Comment:
Can we generalize this check as all the value inside `x_scale_size` is 1? and how about the case when `x_scale_size` is `[1, 1, 1]`?